packages/playground/website/src/components/browser-chrome/style.module.css

```diff
@@ -53,6 +55,7 @@ body.is-embedded .fake-window-wrapper {
 	overflow: hidden;
 
 	animation: pulse 6s ease-in infinite;
+	transition: all ease-in 0.25s;
```

Comment:
This makes transitions happen on page resize as well. It makes the full page window size seem laggy when resizing. It would be nice if this transition did not happen when the viewport changes size.

https://github.com/WordPress/wordpress-playground/assets/841763/a201b16f-0e4d-4a52-bd93-34b6f7bf40d9

Reply:
Good catch, agreed!